### ISSUE TITLE
Is this project actively maintained?

### ISSUE THREAD

**Issue by andreak** [petropavel13/pg_rrule#14]
Hi.

First; I'm looking for exactly this so thanks for having written this!

I see no new commits have been pushed for the last few years and the issues don't seem to get addressed(closed). That raises a concern if this project is maintained.

Thanks for any clarification.

**Comment by petropavel13** [comment]
Hi, @andreak! This project is no longer maintained by me. I don't have a time for this, coz I've switched from backend development to mobile development. 
I can give repository permissions for people who want to maintain this project.

Thanks for being interested!

**Comment by andreak** [comment]
I see several forks of this project. Would be cool if we could collaborate instead of forking...
It'd be great if you gave me permissions to this repo. We're about to implement recurring-events in our application and would like to calculate the recurring-events for a given period in the database. The time-schedule for this is q1-2018.

**Comment by davisford** [comment]
@andreak I realize this is old, but did you find a proper fork to settle on for folks that want to continue this effort?

**Comment by andreak** [comment]
@davisford No, but took a quick look and it seems @ondrej-111's fork is kind of active: https://github.com/ondrej-111/pg_rrule
Maybe we should contact him.

**Comment by thomastthai** [comment]
@andreak, did you make additions or bug fixes to the code?

**Comment by Natureshadow** [comment]
See #22
